src/prezzemolo/graph.py

```diff
@@ -87,6 +88,39 @@ def breadth_first_search(self, start: Vertex[ValueType], end: Vertex[ValueType])
         # start and end are type checked inside _breadth_first_search()
         return self._breadth_first_search(start, end, vertex_2_parent=None)
 
+    def _dijkstra_search(
+        self, start: Vertex[ValueType], end: Optional[Vertex[ValueType]], vertex_2_parent: Optional[Dict[Vertex[ValueType], Optional[Vertex[ValueType]]]]
+    ) -> bool:
+        distances: Dict[Vertex[ValueType], float] = {v: float("inf") for v in self.__vertexes}
+        distances[start] = 0.0
+        priority_queue: List[Tuple[float, Vertex[ValueType]]] = [(0.0, start)]
```

Comment:
While we are using this list as a priority queue, I suggest giving this variable a name that is descriptive of its function within the algorithm: something like `remaining` or `remaining_vertexes`.

Reply:
Also could you use a NamedTuple instead of a Tuple? You could call it `_ShortestDistance` or something like that.